scriptoocloud/src/main/webapp/WEB-INF/views/admin/user/list.html

```diff
@@ -24,6 +24,14 @@ <h2>Users</h2>
 			<form th:if="${user.admin}" th:action="@{|/admin/user/${user.username}/admin/remove|}" method="POST">
 				<button style="padding:0px" class="btn btn-link"><i class="fas fa-user"></i> Remove Admin Role</button>
 			</form>
+
+			<form th:if="${not user.getRoles().toString().contains('ROLE_MANAGER')}" th:action="@{|/admin/user/${user.username}/role/add?roles=ROLE_MANAGER|}" method="POST">
```

Comment:
that's pretty error prone, what if there is another role `ROLE_MANAGER_USER` or something? Thymeleaf should be able to let you do a contains directly on the list: https://www.baeldung.com/thymeleaf-lists-utility